distribution/debian/install.sh

```diff
@@ -45,22 +74,50 @@ app_bin=${app^}                # Binary Name of the app
 
 # This script should not be ran from installdir, otherwise later in the script the extracted files will be removed before they can be moved to installdir.
 if [ "$installdir" == "$(dirname -- "$( readlink -f -- "$0"; )")" ] || [ "$bindir" == "$(dirname -- "$( readlink -f -- "$0"; )")" ]; then
-    echo "You should not run this script from the intended install directory. The script will exit. Please re-run it from another directory"
+    echo ""
+    echo -e "${red}Error!${reset} You should not run this script from the intended install directory."
+    echo "Please re-run it from another directory."
+    echo "Exiting Script!"
     exit
 fi
 
+# User warning about permission conflicts
+echo ""
+echo -e ${red}"           WARNING! WARNING! WARNING!"${reset}
+echo ""
+echo -e "         It is ${red}CRITICAL${reset} that the ${brown}User${reset} and ${brown}Group${reset} you select"
+echo -e "       to run ${brown}[${app^}]${reset} will have both ${red}READ${reset} and ${red}WRITE${reset} access"
+echo -e "      to your Media Library and Download Client directories!"${reset}
+
 # Prompt User
-read -r -p "What user should ${app^} run as? (Default: $app): " app_uid < /dev/tty
+echo ""
+read -r -p "What user should [${app^}] run as? (Default: $app): " app_uid
```

Comment:
why have the `< /dev/tty` inputs been removed?
They were needed to make the script work when using it piped via curl